aws/resource_aws_transfer_server_test.go

```diff
@@ -0,0 +1,158 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/transfer"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSTransferServer_basic(t *testing.T) {
+	var conf transfer.DescribedServer
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
Sorry one more item, each of this resource's `TestCase` should implement a call to a `CheckDestroy` function to verify the Transfer Server was actually removed in AWS: https://www.terraform.io/docs/extend/testing/acceptance-tests/testcase.html#checkdestroy

Something like this should get you started:

```go
func testAccCheckAWSTransferServerDestroy(s *terraform.State) error {
  conn := testAccProvider.Meta().(*AWSClient).transferconn

  for _, rs := range s.RootModule().Resources {
    if rs.Type != "aws_transfer_server" {
      continue
    }

    _, err := conn.DescribeServer(&transfer.DescribeServerInput{
      ServerId: aws.String(rs.Primary.ID),
    })

    if isAWSErr(err, transfer.ErrCodeResourceNotFoundException, "") {
      continue
    }

    if err != nil {
      return err
    }
  }

  return nil
}
```

Reply:
Thanks. I added this method.